### DISCUSSION TITLE
Shopware plugin

### DISCUSSION THREAD

**Discussion by ostark** [spatie/ray#229] [Ideas]
Since you try to cover the whole PHP ecosystem 🤯, a [Shopware](https://github.com/shopware) plugin might be useful as well. 

Unfortunately, I can not help with that - all I know: it is based on Symfony, Twig and the [Builtwith.com graph](https://trends.builtwith.com/shop/Shopware) looks promising.

**Comment by freekmurze** [comment]
Thanks for suggesting this, but we don't have any experience with Shopware either.
